client_cli/Cargo.toml

```diff
@@ -26,3 +26,7 @@ color-eyre = "0.6.2"
 clap = { version = "3.2.16", features = ["derive"] }
 dialoguer = { version = "0.10.2", default-features = false }
 serde_json = "1.0.83"
+
+[build-dependencies]
+anyhow = "1.0.60"
+vergen = { version = "7.3.2", default-features = false, features = ["git"] }
```

Comment:
this is the first time I see `vergen` used. Could we use it in other build scripts?

Reply:
@mversic, actually we already use `vergen` in `telemetry` (`telemetry/build.rs`).

Reply:
not very relevant, but do you think `client/build.rs` would benefit from it?

Reply:
currently we are using `client/build.rs` for building smartcontracts, where do you want to include version inside `client`?